LbhNotificationsApi.Tests/V1/E2ETests/NotificationE2EDynamoDbTest.cs

```diff
@@ -114,6 +115,7 @@ public async Task GetEntityByIdNotFoundReturns404()
         {
             var id = Guid.NewGuid();
             var uri = new Uri($"api/v2/notifications/{id}", UriKind.Relative);
+            Client.DefaultRequestHeaders.Authorization = AuthenticationHeaderValue.Parse(Token);
```

Comment:
I think we should move it to basic class:

![image](https://user-images.githubusercontent.com/23430041/146139480-b8f5d347-8c0f-4f06-855a-202e1affc7f4.png)

Reply:
ok